app/commands/stories/mark_as_read.rb

```diff
@@ -7,6 +7,6 @@ def initialize(story_id, repository = StoryRepository)
   end
 
   def mark_as_read
-    @repo.fetch(@story_id).update_attributes(is_read: true)
+    @repo.fetch(@story_id).update(is_read: true)
```

Comment:
I think I might use `update!` for these so that it will fail loudly when it doesn't work for some reason.

Reply:
Yes, good idea.